File: src/models/Question.js
export class Question {
  constructor(data = {}) {
    this.id = data.id || null;
    this.section = data.section || '';
    this.question = data.question || '';
    this.optionA = data.optionA || '';
    this.optionB = data.optionB || '';
    this.optionC = data.optionC || '';
    this.optionD = data.optionD || '';
    this.correctOption = data.correctOption || '';
  }
}

File: src/main/java/com/jobportel/boot/model/Question.java
package com.jobportel.boot.model;

import lombok.Data;

@Data
public class Question {
    private Long id;
    private String section;
    private String question;
    private String optionA;
    private String optionB;
    private String optionC;
    private String optionD;
    private String correctOption;
}